cts-java/src/test/java/org/ga4gh/cts/api/variants/VariantsMethodsEndpointAliveIT.java

```diff
@@ -0,0 +1,71 @@
+package org.ga4gh.cts.api.variants;
+
+import org.ga4gh.ctk.CtkLogs;
+import org.ga4gh.ctk.transport.GAWrapperException;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.Utils;
+import org.ga4gh.methods.SearchCallSetsRequest;
+import org.ga4gh.methods.SearchVariantsRequest;
+import org.ga4gh.methods.SearchVariantsResponse;
+import org.ga4gh.models.CallSet;
+import org.ga4gh.models.Variant;
+import org.ga4gh.models.VariantSet;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.net.HttpURLConnection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Verify basic operations on variants, variantsets, and callsets.
+ */
+@Category(VariantsTests.class)
+public class VariantsMethodsEndpointAliveIT implements CtkLogs {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     * Check that searching {@link Variant}s with a nonexistent reference name returns
```

Comment:
I recall the thread you're referring to, and I believe this was in the context of a freeform metadata search for SearchReadGroupSets etc. This case is subtly different - `referenceName` is being used as an alias for a `Reference` within the associated `ReferenceSet`. This could debatably be switched with `referenceId`, or augmented with the option to pass a `referenceId` instead in which case you surely would expect a 404 for a non-existent `Reference`.

In the other case of freeform criteria search, `name` does not refer to any existing resource - it's just filtering on an attribute on the target resource.

I think we can call the specification ambiguous at best on this point, and I don't think the current behavior is a conservative assumption so I'd prefer this test were just removed for now. FYI: Google's API will pass this test case as you have it today, though I personally would prefer that it didn't.